CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ When you're ready, submit a pull request!
 
 * As with any community interaction, you must follow the [Code of Conduct](./CODE_OF_CONDUCT.md).
 
-* All changes must have adequate test coverage. For instruction on writing and running the various test suites, see Testing your changes.
+* All changes must have adequate test coverage. For instructions on writing and running the various test suites, see "docs/testing_your_changes.md" in each repository.
```

Comment:
**thought:** I think that we could rephrase this in a way that points to each repositories contribution guidelines. In openfga/openfga we'd have a toplevel `CONTRIBUTING.md` that links to org-level contribution guidelines, and goes into details around contributing to that specific code base.

Reply:
I agree with this 💯

Reply:
Ok, updated